src/visitor/RewriteVisitor.h

```diff
@@ -14,23 +14,45 @@
 namespace nebula {
 namespace graph {
 
-class RewriteMatchLabelVisitor final : public ExprVisitorImpl {
+using Matcher = std::function<bool(const Expression*)>;
+using Rewriter = std::function<Expression*(const Expression*)>;
```

Comment:
Move the typedef into RewriteVisitor to avoid the whole namespace exposation.

Reply:
Fixed.